templates/csharp/copilot-plugin-from-scratch/Program.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
+    .ConfigureServices(services =>
+    {
+        services.AddApplicationInsightsTelemetryWorkerService();
```

Comment:
Do we create application insights for user?

Reply:
Removed all configurations for application insights, thanks!